### PR TITLE
[CHORE] Bump version 3.0.0

### PR DESCRIPTION
**Pull request by JiahuiWho**
For release 3.0.0

### CODE REVIEW

**Comment by lijamie98** [comment]
[ANCHOR-431] Set callback-api-request.enabled default to false is a breaking change. Can we make sure the PR is included in v3.0?

File: .github/workflows/sub_essential_tests.yml [hunk 1]
```diff
@@ -11,7 +11,7 @@ jobs:
     steps:
       #############################################
       # Setup JDK 17
-      # Download, and Extract java-stellar-anchor-sdk.tar
+      # Download, and Extract anchor-platform.tar
```
Comment (lijamie98) [inline .github/workflows/sub_essential_tests.yml:14]: Thans for handling the GH repo rename

File: docs/README.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 [![License](https://badgen.net/badge/license/Apache%202/blue?icon=github&label=License)](https://github.com/stellar/java-stellar-anchor-sdk/blob/develop/LICENSE)
 [![GitHub Version](https://badgen.net/github/release/stellar/java-stellar-anchor-sdk?icon=github&label=Latest%20release)](https://github.com/stellar/java-stellar-anchor-sdk/releases)
-[![Docker](https://badgen.net/badge/Latest%20Release/v2.11.1/blue?icon=docker)](https://hub.docker.com/r/stellar/anchor-platform/tags?page=1&name=2.11.1)
+[![Docker](https://badgen.net/badge/Latest%20Release/v3.0.0/blue?icon=docker)](https://hub.docker.com/r/stellar/anchor-platform/tags?page=1&name=3.0.0)
```
Comment (lijamie98) [inline docs/README.md:3]: This indicate version `2.12.0` was not back-merged to `develop` branch. I will submit another PR for `2.12.0` before v3 is merged.

**Review by lijamie98 — CHANGES_REQUESTED** [review]

**Comment by lijamie98** [comment]
Blocking the PR for now until the concerns are taken care of.

**Comment by JiahuiWho** [comment]
2.12 back merge https://github.com/stellar/anchor-platform/pull/1597

File: docs/README.md (hunk 2 above)
Comment (JiahuiWho) [reply docs/README.md:3]: https://github.com/stellar/anchor-platform/pull/1597

**Review by philipliu — APPROVED** [review]

**Review by lijamie98 — APPROVED** [review]

**Pull request merged**